python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
   - name: u_calib_photometry_used
     "@id": "#Object.u_calib_photometry_used"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1154
   - name: u_calib_psf_candidate
     "@id": "#Object.u_calib_psf_candidate"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1155
   - name: u_calib_psf_reserved
     "@id": "#Object.u_calib_psf_reserved"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1156
   - name: u_calib_psf_used
     "@id": "#Object.u_calib_psf_used"
     datatype: boolean
     description: Propagated from sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1157
   - name: u_centroid_flag
     "@id": "#Object.u_centroid_flag"
     datatype: boolean
     description: General Failure Flag. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1158
   - name: u_centroid_x
     "@id": "#Object.u_centroid_x"
     datatype: double
     description: Centroid from SDSS Centroid algorithm. Measured on u-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 1159
   - name: u_centroid_xErr
     "@id": "#Object.u_centroid_xErr"
     datatype: float
     description: 1-sigma uncertainty on x position. Measured on u-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 1160
   - name: u_centroid_y
     "@id": "#Object.u_centroid_y"
     datatype: double
     description: Centroid from SDSS Centroid algorithm. Measured on u-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 1161
   - name: u_centroid_yErr
     "@id": "#Object.u_centroid_yErr"
     datatype: float
     description: 1-sigma uncertainty on y position. Measured on u-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 1162
   - name: u_decl
     "@id": "#Object.u_decl"
     datatype: double
     description: Position in Declination. Measured on u-band.
     fits:tunit: deg
     ivoa:ucd: pos.eq.dec
+    tap:principal: 0
+    tap:column_index: 1002
   - name: u_extendedness
     "@id": "#Object.u_extendedness"
     datatype: double
     description: Set to 1 for extended sources, 0 for point sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 1
+    tap:column_index: 1163
   - name: u_extendedness_flag
     "@id": "#Object.u_extendedness_flag"
     datatype: boolean
     description: Set to 1 for any fatal failure. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1164
   - name: u_free_cModelFlux
     "@id": "#Object.u_free_cModelFlux"
     datatype: double
     description: Flux from the final cmodel fit. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1165
```

Comment:
The "free" versions of CModel and PSF fluxes should probably go right after their not-free variants, instead of appearing alphabetically.  I'm also baffled that we don't have free versions of any other CModel parameters (though it seems a lot of those unfortunately lose their CModel prefix in standardization process).